xml/rmt_install.xml

```diff
@@ -219,638 +220,721 @@
-       &rmt; service will be accessible from clients
-      </para>
-     </listitem>
-    </varlistentry>
-    <varlistentry>
-     <term><emphasis role="strong">Key</emphasis>: ingress.tls[0].hosts[0]</term>
-     <listitem>
-      <para>
-       <emphasis role="strong">Type</emphasis>: string
-      </para>
-      <para>
-       <emphasis role="strong">Default</emphasis>:
-       <literal>"chart-example.local"</literal>
-      </para>
-      <para>
-       <emphasis role="strong">Description</emphasis>: DNS name at which the
-       &rmt; service will be accessible from clients
-      </para>
-     </listitem>
-    </varlistentry>
-    <varlistentry>
-     <term><emphasis role="strong">Key</emphasis>: ingress.tls[0].secretName</term>
-     <listitem>
-      <para>
-       <emphasis role="strong">Type</emphasis>: string
-      </para>
-      <para>
-       <emphasis role="strong">Default</emphasis>:
-       <literal>"rmt-cert"</literal>
-      </para>
-      <para>
-       <emphasis role="strong">Description</emphasis>:
-      </para>
-     </listitem>
-    </varlistentry>
-    <varlistentry>
-     <term><emphasis role="strong">Key</emphasis>: serviceAccount.annotations</term>
-     <listitem>
-      <para>
-       <emphasis role="strong">Type</emphasis>: object
-      </para>
-      <para>
-       <emphasis role="strong">Default</emphasis>: <literal>{}</literal>
-      </para>
-      <para>
-       <emphasis role="strong">Description</emphasis>:
-      </para>
-     </listitem>
-    </varlistentry>
-    <varlistentry>
-     <term><emphasis role="strong">Key</emphasis>: serviceAccount.create</term>
-     <listitem>
-      <para>
-       <emphasis role="strong">Type</emphasis>: bool
-      </para>
-      <para>
-       <emphasis role="strong">Default</emphasis>: <literal>true</literal>
-      </para>
-      <para>
-       <emphasis role="strong">Description</emphasis>:
-      </para>
-     </listitem>
-    </varlistentry>
-    <varlistentry>
-     <term><emphasis role="strong">Key</emphasis>: serviceAccount.name</term>
-     <listitem>
-      <para>
-       <emphasis role="strong">Type</emphasis>: string
-      </para>
-      <para>
-       <emphasis role="strong">Default</emphasis>: <literal>""</literal>
-      </para>
-      <para>
-       <emphasis role="strong">Description</emphasis>:
-      </para>
-     </listitem>
-    </varlistentry>
-   </variablelist>
+   <screen>cat &lt;&lt; EOF > myvalues.yaml
+---
+app:
+scc:
+   username: UXXXXXXX
+   password: PASSXXXX
+   products_enable:
+      - SLES/15.3/x86_64
+      - sle-module-python2/15.3/x86_64
+   products_disable:
+      - sle-module-legacy/15.3/x86_64
+      - sle-module-cap-tools/15.3/x86_64
+ingress:
+enabled: true
+hosts:
+   - host: chart-example.local
+      paths:
+      - path: "/"
+         pathType: Prefix
+tls:
+- secretName: rmt-cert
+   hosts:
+   - chart-example.local
+EOF
+> helm install rmtsle oci://registry.suse.com/suse/rmt-helm -f myvalues.yaml</screen>
```

Comment:
I would make the `helm install` command a `<screen>` block of its own - otherwise, it gets sort of "lost" as the last line of this block. 
Something like...

```suggestion
   </screen>

   <para>
    And to install RMT, run:
   </para>
   <screen>
> helm install rmtsle oci://registry.suse.com/suse/rmt-helm -f myvalues.yaml</screen>
```

Reply:
Hi @ngetahun why is this comment resolved?